### PR TITLE
New repo for Beginners

### PR DESCRIPTION
**Pull request by muneersyed156**
## Contributing

If you want to add a project to this list, please make sure that:

- [x] You read the [contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2020/blob/master/CONTRIBUTING.md)
- [x] The project can't be just an "add your name" or simply + 1 contribution to hacktoberfest
- [x] Place them in alphabetical order
- [x] One repository per PR/contribution - _avoid conflicts and simplify review_
- [x] Your pull request is not spammy just adding an empty line, otherwise it will be marked as invalid and not count to hacktoberfest.

If they meet the requirements above, feel free to create a PR. Happy Hacking! :sparkles:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,10 @@
 - [FreeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp)
 - [Funny Algorithms](https://github.com/ReciHub/FunnyAlgorithms)
 - [Hackerrank-codes](https://github.com/swapnanildutta/Hackerrank-Codes)
+- [HacktoberFest2020-Contributions](https://github.com/bhaveshlohana/
```
Comment (OtacilioN) [inline README.md:72]: I think we got some conflict here, can you fix it?

**Review by OtacilioN — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (muneersyed156) [reply README.md:72]: May I know what happened?

**Comment by muneersyed156** [comment]
Resolved the conflict can you check it once!

**Review by OtacilioN — APPROVED** [review]
Awesome, thanks for contributing!

**Pull request merged**